react/ui-components/src/hoc/MultiChildFormWrapper.js

```
@@ -0,0 +1,91 @@
+              }}
+              title="Remove"
+            >
+              ✕
+            </div>
+          )}
+
+          {config.body.map((field, i) => {
+            const fieldName = `${config.name}.${idx}.${field.populators.name}`;
+            return (
+              <div style={{marginBottom:"1rem"}}>
+              <FieldController
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)